lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -114,9 +115,43 @@ if test -f "$ETC_PROVISIONER_JSON"; then
       exit 0
     fi
   fi
+
+  # Skip the repository checks if they are defined in provisioner.json:
+  # there might be non-default (and/or network) path
+  for repo in SLE-Cloud SUSE-Cloud-5-Pool SUSE-Cloud-5-Updates; do
+    # json_read is part of crowbar package
+    common_check="$( json_read $PROVISIONER_JSON attributes.provisioner.suse.autoyast.repos.common.${repo//./\\\\.}.url )"
+    sles11_check="$( json_read $PROVISIONER_JSON attributes.provisioner.suse.autoyast.repos.suse-11\\.3.${repo//./\\\\.}.url )"
+
+    if [ -n "$common_check" -o -n "$sles11_check" ]; then
+      REPOS_SKIP_CHECKS+=" ${repo#SLE-}"
+    fi
+  done
 fi
 
 
+### Check if repositories are prepared for the upgrade
+# We need to ensure that Cloud5 main repository and update repos are in place
+
+check_media_content \
+    Cloud \
+    /srv/tftpboot/suse-11.3/repos/Cloud \
+    1558be86e7354d31e71e7c8c2574031a
+
+# FIXME repos that do not exist yet
+REPOS_SKIP_CHECKS+=" SUSE-Cloud-5-Pool SUSE-Cloud-5-Updates"
```

Comment:
You need to add Cloud to this, see https://github.com/crowbar/crowbar/pull/2081/files#diff-a0b6391cfbad9eb69d24f015b0247548R543